Dockerfile

```diff
@@ -1,11 +1,21 @@
-FROM python:3.8-slim
+FROM python:3.10-slim
 
 WORKDIR /app
 
 COPY . .
 
 ARG HEROKU
-ENV HEROKU $HEROKU
+ENV HEROKU=$HEROKU
+ENV BOT_TOKEN=
+ENV SPOTIFY_ID=
+ENV SPOTIFY_SECRET=
+ENV BOT_PREFIX=
```

Comment:
I initially added default values cause it then shows them automaticly by most docker guis:
![image](https://user-images.githubusercontent.com/28807283/211190692-39ce2a0b-5c43-4038-bfc4-7623f3326f91.png)
if no values where given, you had to manually delete all automaticly generated vars, cause otherwise an error gets thrown:
![image](https://user-images.githubusercontent.com/28807283/211190783-b150eedf-d6d5-4b14-b7d5-5b2fee3dc97c.png)
the defaultvalues are getting overwriten if given at start of the container

Reply:
Hmm, I see. I've removed defaults from here to avoid duplication, but if that's the case I can add it. But can you explain how some empty values are ok and some aren't?

Reply:
I just estimate that the spotify vars are way more likely be used than the rest😅 and the BOT_TOKEN has to be set anyways...

Reply:
Can we omit the error if we put it like `ENV SPOTIFY_ID=""`?

Reply:
Nope. But i think that it's not that necessary cause you can still remove the var's in the gui... same thing with heroku.

Reply:
That's unfortunate